plugins/modules/k8s.py

```diff
@@ -193,6 +193,11 @@
     type: list
     elements: str
     version_added: 3.0.0
+  subresource:
+    description:
+      - Provide the C(subresource) to run your definition against.
+    type: str
+    version_added: 6.1.0
```

Comment:
Please update to 6.2.0, as version 6.1.0 has already been released

Reply:
I've updated the version and also signed the commit.